e2e/angular-core/src/ng-add.test.ts

```diff
@@ -438,8 +438,8 @@ describe('convert Angular CLI workspace to an Nx workspace', () => {
     // add other projects
     const app1 = uniq('app1');
     const lib1 = uniq('lib1');
-    runCommand(`ng g @schematics/angular:application ${app1}`);
-    runCommand(`ng g @schematics/angular:library ${lib1}`);
+    runCommand(`ng g @schematics/angular:application ${app1} --no-interactive`);
+    runCommand(`ng g @schematics/angular:library ${lib1} --no-interactive`);
```

Comment:
while testing I kept getting prompted for adding routing/using stand alone components from this spec file. added to prevent tests from prompting user input for local testing (I assume ng cli turns off prompting for CI, which is why this has been working?)

Reply:
Yeah I think CI forces `--no-interactive`